authorized_keys

```diff
@@ -0,0 +1,2 @@
+command="export GL_ID=key-gitlabshelladmin;if [ -n \"$SSH_ORIGINAL_COMMAND\" ]; then eval \"$SSH_ORIGINAL_COMMAND\";else exec \"$SHELL\"; fi" ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDOg9xXsXZSY4HzS4+nOIUCBqh5HvcdZqYys8ClCeJR0s3mrDucSrx3+IVjoWFkE8utk/W7DLXHo9ayC+gB4jpsd/F/0+b4zIuaypk+Qih/IwGx6zdsDOz0/sht7GzhVCV2HjP11iM1U38wXKMU0kmBR6fG3iqv3TC+nZFScuHg7DC1Q9dyPQf8TrF0JzPP7oKmybYDITGBSl9YaE+XllB3RDI3SiIohFaPSxpPWBxLcLLgIGRKXA+HE7sNBRE2pnvKKI8NvU3ks95jpPTC5OFQnLzLfZke3YfWkKjRhtpxxIyuXNSSkyr9sPZYiGkjMpcRGbMJ8pebdLkhLRxQl63x gitlab_shell@feedhenry
+command="/home/git/gitlab-shell/bin/gitlab-shell key-repoadmin",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC7ghi9VDEdxN9VNxoq6aEg7V7QpN81VBAydaI9JqFEuyX21EZCMj+WyPH80upwBn35huyqYpmKvDSrUWSiRqc2isXSPf/rFBw6KhGMOrNd8MH9HdLg3Rlg/EH3V2TkyyFzQ1JSZswBgk9tFJHPpKsRT6Z+P7dP9m7B/j9tZ9aFiDNhInW9mHKstNUP3i4gtuig4p1uw3UaQGzaIonEGBDE1qele+R5nPowlFdPxbKk99nRGTjkk4Vj7R5t7pjXdtCca62Le0I59AlNOo8ry33J5KTCHcX+kGUPWvQIB8b6Kt3iuoh87kIZwPWkW2p7y39fYBBANxYcwlhAnjuW5mpl repoadmin@feedhenry
```

Comment:
@david-martin Is this file complete as-is, or is there anything that will try to add additional keys at runtime?